inference.py

```diff
@@ -309,9 +267,12 @@ def main(params: dict):
         with tqdm(list_img, desc='image list', position=0) as _tqdm:
             for info in _tqdm:
                 img_name = Path(info['tif']).name
-                local_gpkg = info['gpkg']
+                local_gpkg = info['gpkg'] if 'gpkg' in info.keys() else None
```

Comment:
I would right away convert it to Path:
`Path(info['gpkg']) if 'gpkg' in info.keys() else None`